src/Processors/Executors/ExecutorTasks.cpp

```diff
@@ -53,6 +53,17 @@ void ExecutorTasks::tryGetTask(ExecutionThreadContext & context)
     {
         std::unique_lock lock(mutex);
 
+    #if defined(OS_LINUX)
```

Comment:
Missing a comment about OS_LINUX.

Reply:
Not only, about #ifdef but also about the specific case it handles